RecoBTag/CTagging/src/CharmTagger.cc

```diff
@@ -0,0 +1,149 @@
+#include "RecoBTag/CTagging/interface/CharmTagger.h"
+
+#include "DataFormats/BTauReco/interface/CandSoftLeptonTagInfo.h"
+#include "DataFormats/BTauReco/interface/CandIPTagInfo.h"
+#include "DataFormats/BTauReco/interface/SecondaryVertexTagInfo.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "CondFormats/DataRecord/interface/GBRWrapperRcd.h"
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <map>
+#include <iostream>
+
+#include "TDirectory.h" //DEBUG
+
+CharmTagger::CharmTagger(const edm::ParameterSet & configuration):
+	sl_computer_(configuration.getParameter<edm::ParameterSet>("slComputerCfg")),
+	vars_definition_(configuration.getParameter<vpset>("variables")),
+  mva_name_( configuration.getParameter<std::string >("mvaName") ),
+  use_condDB_(configuration.existsAs<bool>("useCondDB") ? configuration.getParameter<bool>("useCondDB") : false),
```

Comment:
oh, well, the fillDescriptions implementation is not that easy, I guess; if it needs to be propagated to the btaggers framework